msg/vehicle_odometry.msg

```diff
@@ -29,7 +29,10 @@ float32 y			# East position
 float32 z			# Down position
 
 # Orientation quaternion. First value NaN if invalid/unknown
-float32[4] q			# Quaternion rotation from NED earth-fixed frame to XYZ body frame
+float32[4] q			# Quaternion rotation from FRD body frame to fixed frame
+
+float32[4] q_err		# Quaternion rotation from odometry's fixed frame to local_origin_frame
```

Comment:
I don't like this name. @jkflying @TSC21 any other naming ideas for this?

Reply:
`q_odom`?

Reply:
What about q_offset?